packages/connectors/react/src/inject.jsx

```diff
@@ -31,8 +31,18 @@ export default injectFunction => WrappedComponent => class extends Component {
   }
 
   componentWillMount() {
+    const { store } = this.context
+
+    if (!store) {
+      // eslint-disable-next-line no-console
+      console.error('To use the inject function, one of the parents of this component needs to be a `provider`.\n' +
```

Comment:
I like the idea of :
 - Show what is the component that need to be fixed
 - Give a link to the documentation
 - Give an hint (use `provider`)

I have the feeling that:
 - we need to say "hey! The error comes from a k-ramel component!"
 - this is not quite exact to say a parents should be a `provider` (the important part is that the store is in the context, you can use our HoC or a Component to provide this, like `Provider` from `react-redux`).

What do you think about this?
```
[@k-ramel/react] Error in `inject` the Component for onlyUpdateForPropTypes(Header)

The store needs to be provided by an ancestor of this component.
> You can use `provider` from `@k-ramel/react` or `Provider` from react-redux.

Check the documentation for an example at https://github.com/alakarteio/k-ramel#connect-it-with-reactjs
```

Reply:
I'm agree, it's better like this.

Reply:
```
[@k-ramel/react] Error while wrapping `inject` to the Component onlyUpdateForPropTypes(Header)
/// ....
 ```